### PR TITLE
fix: request twice when ignore is reg

### PR DESCRIPTION
**Pull request by popomore**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - "5"
+  - "7"
+  - "6"
   - "4"
   - "0.12"
```
Comment (fengmk2) [inline .travis.yml]: remove 0.12

Comment (popomore) [inline .travis.yml]: it's a patch, should support 0.12.

Comment (fengmk2) [inline .travis.yml]: 0.12 need --harmony flag

**Comment by popomore** [comment]
I have fixed on 0.12

**Comment by fengmk2** [comment]
1.3.1

**Pull request merged**